backend/src/main/resources/application-local.yml

```
@@ -5,15 +5,15 @@ spring:
   datasource:
     generate-unique-name: false
   jpa:
-    show-sql: true
     properties:
       hibernate:
         format_sql: true
+        highlight_sql: true
         ddl-auto: create
     defer-datasource-initialization: true
```

```suggestion
        format_sql: true
        highlight_sql: true
        ddl-auto: create
        use_sql_comments: false
    defer-datasource-initialization: true
```

이렇게 해서 평소엔 꺼두다가, 필요할 때 true로 바꿔 사용하면 어때요?

---

좋아용 👍